product_kosher/models/product_template.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 OpenSynergy Indonesia
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import models, fields, api
+from datetime import datetime
+
+_KOSHER_STATE = [
+    ('certified', 'Certified'),
+    ('not_certified', 'Not Certified')
+]
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    @api.multi
+    @api.depends(
+        "kosher_ids",
+        "kosher_ids.name",
+        "kosher_ids.date_start",
+        "kosher_ids.date_end",
+    )
+    def _compute_kosher_state(self):
+        date_now = datetime.now().strftime("%Y-%m-%d")
+        for product in self:
+            product.kosher_id = False
```

Comment:
I think this line can be removed. All conditions of your if/else chains below also set `.kosher_id`, so this is generating a duplicate write